base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/dialogs/RecentConfigSelectionDialog.java

```diff
@@ -0,0 +1,55 @@
+package uk.ac.stfc.isis.ibex.ui.configserver.dialogs;
+
+import java.util.Collection;
+
+import org.eclipse.swt.widgets.Shell;
+
+import uk.ac.stfc.isis.ibex.configserver.configuration.ConfigInfo;
+
+/**
+* Dialog for asking the user to select a single recent configuration or component.
+*/
+public class RecentConfigSelectionDialog extends MultipleRecentConfigsSelectionDialog {
```

Comment:
This feels like duplication. Can you get `MultipleRecentConfigsSelectionDialog`  to work when you only have one config?